lineapy/utils/config.py

```diff
@@ -1,7 +1,7 @@
 import logging
 from pathlib import Path
 
-FOLDER_NAME = ".linea"
+FOLDER_NAME = ".lineapy"
 
 LOG_FILE_NAME = "linea.log"
```

Comment:
Should we change the log file name too? to lineapy.log?

Reply:
we could.. i think the main concern was matching the folder name that sits in user's home to our package. lineapy can be easily spotted and tell the users that everything under it belong to this new awesome library you just installed.

Reply:
noticed that we have merged so not that big a deal anymore.